tachyon/c/examples/prime_field.cc

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+
+#include "tachyon/c/math/elliptic_curves/bn/bn254/fr.h"
+
+int main() {
+  // Initialize prime field elements
+  tachyon_bn254_fr a = tachyon_bn254_fr_random();
+  tachyon_bn254_fr b = tachyon_bn254_fr_random();
+
+  // Display the random elements
```

Comment:
`a` and `b` are not displayed currently - can this be added?

Reply:
Do you mean print them on log? It would be difficult cuz there is no ToString() in C API ig. Or I can fix this comment.